docs/security/linode-manager-security-controls-new-manager/index.md

```diff
@@ -0,0 +1,260 @@
+
+![Diagram of two-factor authentication with SSH login.](two-factor-authentication-diagram.png "Two-factor authentication with SSH login.")
+
+If you enable this optional feature in the Linode Manager, you'll access your Linode Manager account using your smartphone as a physical token in addition to your username and password. This additional layer of security reduces the risk that an unauthorized individual will gain access to your Linode Manager account.
+
+### Select a Token Application
+
+Before you enable two-factor authentication in the Linode Manager, select a token application for your smartphone. This guide will use Authy as an example, but you can use any application that supports the Time-based One-Time Password (TOTP) algorithm. For example, you can use any of the following applications:
+
+* [Authy](https://authy.com/features/setup/) (iOS/Android/Chrome)
+* [Google Authenticator](http://support.google.com/accounts/bin/answer.py?hl=en&answer=1066447) (iOS/Android/BlackBerry)
+* [Duo Mobile](http://guide.duosecurity.com/third-party-accounts) (iOS/Android)
+
+Install one of these applications on your smartphone before continuing.
+
+{{< note >}}
+Authy stores your authentication tokens (hashed for security) on their servers. This makes it possible for them to support backing up and restoring tokens, as well as making it easy to switch devices. However, some users may not be comfortable storing sensitive information in the cloud; for these users, Google Authenticator is a better choice, as the authentication keys are only stored locally.
+{{< /note >}}
+
+### Enable Two-Factor Authentication
+
+Enable two-factor authentication to start using it with your Linode Manager account.
+
+1.  Log in to the [Linode Manager](https://manager.linode.com).
+2.  Select the **My Profile** link by clicking on your username at the top of the page:
+
+   ![Select 'My Profile'](security-my-profile.png "Select 'My Profile')
+
+3.  Select the **Password & Authentication** tab.
+4.  In the *Two-Factor Authentication (TFA)* section, toggle the *Disabled* box so that it reads *Enabled* to enable Two-Factor Authentication.
+5.  The screen shown below appears. Write down the **Secret Key** and store it in a safe place:
+
+    [![The key and QR code for two-factor authentication.](security-tfa-settings-small.png "The key and QR code for two-factor authentication")](security-tfa-settings.png)
+
+    {{< caution >}}
+Do not refresh this page until you have configured Authy on your phone. The key will be shown only once. If you get locked out of your account, [contact support](/docs/support) to regain access.
```

Comment:
This doesn't apply in the new manager. We don't turn two-factor on for your account unless you confirm your token. If you reload the page you can just generate a new token and start over.